### PR TITLE
replace pkg/errors

### PR DESCRIPTION
**Pull request by zouyee**
replace `pkg/errors`

Signed-off-by: Zou Nengren <zouyee1989@gmail.com>

### CODE REVIEW

**Review by AkihiroSuda — APPROVED** [review]

File: plugin/plugin.go [hunk 1]
```diff
@@ -52,7 +54,7 @@ func init() {
 
 			snapshotter, err := aufs.New(root)
 			if err != nil {
-				return nil, errors.Wrap(plugin.ErrSkipPlugin, err.Error())
+				return nil, fmt.Errorf("%s: %w", err.Error(), plugin.ErrSkipPlugin)
```
Comment (mikebrow) [inline plugin/plugin.go:57]: accurate port... but it feels it feels like the wrap is backwards maybe it should read:
fmt.Errorf("%s: %w", plugin.ErrSkipPlugin.Error(), err)

File: aufs.go [hunk 2]
```diff
@@ -404,7 +403,7 @@ func supported() error {
 			return nil
 		}
 	}
-	return errors.Errorf("aufs is not supported" + probeError)
+	return fmt.Errorf("aufs is not supported" + probeError)
```
Comment (mikebrow) [inline aufs.go]: I think probeError should be an error not a string?
the wrapped sprintf above isn't in the right wrap format `string: %w`  not sure what the benefit was to add out on error to the probeError string...
if those changes are made then this line would be ```fmt.Errorf("aufs is not supported: %w", probeError)```

**Review by mikebrow — COMMENTED** [review]
couple comments.. nothing to do with your changes more the structure of the prior defined errors

File: aufs.go (hunk 2 above)
Comment (zouyee) [reply aufs.go]: Done

File: plugin/plugin.go (hunk 1 above)
Comment (mikebrow) [reply plugin/plugin.go:57]: Apologies, I was wrong you had it right the first time with fmt.Errorf("%s: %w", err.Error(), plugin.ErrSkipPlugin

File: plugin/plugin.go [hunk 3]
```diff
@@ -52,7 +54,7 @@ func init() {
 
 			snapshotter, err := aufs.New(root)
 			if err != nil {
-				return nil, errors.Wrap(plugin.ErrSkipPlugin, err.Error())
+				return nil, fmt.Errorf("%s: %w", plugin.ErrSkipPlugin.Error(), err)
```
Comment (mikebrow) [inline plugin/plugin.go]: Apologies, I was wrong you had it right the first time with fmt.Errorf("%s: %w", err.Error(), plugin.ErrSkipPlugin

Comment (mikebrow) [reply plugin/plugin.go]: the reason is we want to be able to check with is against ErrSkipPlugin and with this error wrapping model you can't nest errors other than with text prefixes..

Comment (zouyee) [reply plugin/plugin.go]: Done

**Review by estesp — APPROVED** [review]
LGTM

**Review by AkihiroSuda — APPROVED** [review]

**Pull request merged**